src/main/java/com/taboola/backstage/model/media/campaigns/Campaign.java

```diff
@@ -181,11 +183,25 @@ public String getComments() {
     }
 
     public Date getStartDate() {
-        return startDate;
+        if (startDate != null) {
+            try {
+                return dateFormat.parse(startDate);
```

Comment:
The pattern is not correct it should be 'yyyy-MM-dd'

Reply:
Changed to 'yyyy-MM-dd'